packages/strapi-upload-aws-s3/package.json

```diff
@@ -43,4 +43,4 @@
     "npm": ">= 5.3.0"
   },
   "license": "MIT"
-}
+}
```

Comment:
I think it's time to configure your editor to add a new line at the end of the file:
If you're using vscode I recommend this defaults

```
"files.trimTrailingWhitespace": true,
"files.insertFinalNewline": true,
"files.trimFinalNewlines": true,
```

Reply:
@kamalbennani My editor is already configured to add one line at the end of files.
I think we have an issue on `pre-commit` for that.